geonode_mapstore_client/client/js/plugins/Operation/components/ExecutionRequestTable.jsx

```diff
@@ -41,7 +42,7 @@ function ExecutionRequestTable({
                     <div className="gn-main-event-content">
                         <div className="gn-main-event-text">
                             <div className="gn-main-icon">
-                                <Icon glyph={iconName}/>
+                                <Glyphicon glyph={iconName}/>
```

Comment:
This iconName is provided as a prop from following components OperationUpload, UploadDataset and UploadDocument. The UploadDataset and UploadDocument are routes and we should use the icon supported by the glyphicon set (dataset and document glyphs). The OperationUpload is used inside the Operation plugin so the configuration should be reviewed inside localConfig

Reply:
@allyoucanmap 

**Metadata upload**
Can we use this alternative near match glyph icon?
| Lib      | icon     |
|---------------|---------------|
| fa | <img width="50" height="50" alt="image" src="https://github.com/user-attachments/assets/3cf6dedd-ffcc-4758-b0c3-c7a32295c698" /> |
| glyphicon | <img width="50" height="50" alt="image" src="https://github.com/user-attachments/assets/476074b8-c9e7-450e-bec8-5640e4419355" /> |

**Style upload**
Icon missing in glyphicon, no alternative found
| Lib      | icon     |
|---------------|---------------|
| fa | <img width="50" height="50" alt="image" src="https://github.com/user-attachments/assets/fddd0c6a-8599-4436-aa7b-6b8c341d51a2" /> |
| glyphicon | - |